docker/lr-align/Makefile

```diff
@@ -1,14 +1,18 @@
-VERSION = 0.1.27
+VERSION = 0.1.28
```

Comment:
while you are here, let's take away the quay repo.

Reply:
👍